inst/include/interface/rcpp/rcpp_interface.hpp

```diff
@@ -337,28 +337,28 @@ RCPP_MODULE(fims) {
   Rcpp::function("clear_selectivity_log", clear_selectivity_log);
   Rcpp::function("clear_debug_log", clear_debug_log);
 
-  Rcpp::class_<Parameter>("Parameter")
+  Rcpp::class_<Parameter>("Parameter", "FIMS Parameter Class")
       .constructor()
       .constructor<double>()
       .constructor<Parameter>()
-      .field("value", &Parameter::value_m)
-      .field("min", &Parameter::min_m)
-      .field("max", &Parameter::max_m)
-      .field("id", &Parameter::id_m)
-      .field("is_random_effect", &Parameter::is_random_effect_m)
-      .field("estimated", &Parameter::estimated_m);
+      .field("value", &Parameter::value_m, "numeric parameter value")
+      .field("min", &Parameter::min_m, "minimum parameter value")
+      .field("max", &Parameter::max_m, "maximum parameter value")
+      .field("id", &Parameter::id_m, "unique id for parameter class")
+      .field("is_random_effect", &Parameter::is_random_effect_m, "boolean indicating whether or not parameter is a random effect; default value is FALSE")
+      .field("estimated", &Parameter::estimated_m, "boolean indicating whether or not parameter is estimated; default value is FALSE");
 
    Rcpp::class_<ParameterVector>("ParameterVector")
       .constructor()
       .constructor<size_t>()
       .constructor<Rcpp::NumericVector, size_t>()
-      .field("data", &ParameterVector::storage_m)
-      .method("at", &ParameterVector::at)
-      .method("size", &ParameterVector::size)
-      .method("resize", &ParameterVector::resize)
-      .method("set_all_estimable", &ParameterVector::set_all_estimable)
-      .method("set_all_random", &ParameterVector::set_all_random)
-      .method("fill", &ParameterVector::fill);
+      .field("data", &ParameterVector::storage_m, "list where each element is a Parameter class")
+      .method("at", &ParameterVector::at, "returns a Parameter at the indicated position given th eindex argument")
+      .method("size", &ParameterVector::size, "returns the size of the Parameter Vector")
+      .method("resize", &ParameterVector::resize, "resizes the Parameter Vector given the provided length argument")
+      .method("set_all_estimable", &ParameterVector::set_all_estimable, "sets all Parameters within vector as estimable")
+      .method("set_all_random", &ParameterVector::set_all_random, "sets all Parameters within vector as estimable")
+      .method("fill", &ParameterVector::fill, "sets the value of all Parameters in the vector with the provided value");
```

Comment:
Would it be useful to have a method for fill_min and fill_max?

Reply:
Added fill_min and fill_max with commit [5b07794](https://github.com/NOAA-FIMS/FIMS/pull/623/commits/5b07794ca15f49393fddccd85f2036274c1c7bc7). @msupernaw, did I implement this correctly?